src/main/kotlin/com/autonomousapps/subplugin/ProjectPlugin.kt

```diff
@@ -126,8 +127,13 @@ internal class ProjectPlugin(private val project: Project) {
 
   /** Has the `com.android.application` plugin applied. */
   private fun Project.configureAndroidAppProject() {
+    val project = this
     val appExtension = the<AppExtension>()
-    appExtension.applicationVariants.all {
+    val ignoredVariantNames = androidIgnoredVariants()
+    val allowedVariants = appExtension.applicationVariants.matching { variant ->
+      !ignoredVariantNames.contains(variant.name)
```

Comment:
do we really want to use `contains()` instead of `==`?

Reply:
As I see because ignoredVariantNames is a collection we can't use `==` to compare those params.
So for clarify, we have set of `ignoredVariantNames` and we should intersect it with `applicationVariants`

Reply:
oh sorry! I wasn't thinking clearly. That's a list 🤦